UnattendResources/Logon.ps1

```diff
@@ -291,43 +292,44 @@ try
     if ($vmwareToolsPath) {
         Install-VMwareTools
     }
-    Run-CustomScript "RunBeforeCloudbaseInitInstall.ps1"

```

Comment:
is okay not to run the hooks, but is not okay to remove cloudbase-init non related functionality